YouTubeClone/YTChatMessageContentView.swift

```diff
@@ -0,0 +1,60 @@
+//
+//  YTChatMessageContentView.swift
+//  YouTubeClone
+//
+//  Created by Sagar Dagdu on 29/06/21.
+//  Copyright © 2021 Stream.io Inc. All rights reserved.
+//
+
+import UIKit
+import StreamChat
+import StreamChatUI
+
+final class YTChatMessageContentView: ChatMessageContentView {
+    override var maxContentWidthMultiplier: CGFloat { 1 }
+    
+    override func createTimestampLabel() -> UILabel {
+        let label = super.createTimestampLabel()
+        label.font = appearance.fonts.footnote
+        label.textAlignment = .center
+        label.textColor = .secondaryLabel
+        return label
+    }
+    
+    override func createTextView() -> UITextView {
+        let textView = super.createTextView()
+        textView.font = appearance.fonts.footnote
+        textView.textColor = .secondaryLabel
+        return textView
+    }
+    
+    override func createAuthorNameLabel() -> UILabel {
+        let label = super.createAuthorNameLabel()
+        label.font = appearance.fonts.footnoteBold
+        return label
+    }
+    
+    override var messageAuthorAvatarSize: CGSize {
+        .init(width: 40, height: 40)
+    }
+    
+    override func layout(options: ChatMessageLayoutOptions) {
+        super.layout(options: options)
+        
+        bubbleThreadMetaContainer.axis = .horizontal
+        
+        let subviews = bubbleThreadMetaContainer.subviews
+        bubbleThreadMetaContainer.removeAllArrangedSubviews()
+        bubbleThreadMetaContainer.addArrangedSubviews(subviews.reversed())
+        
+        let metadataSubviews = metadataContainer?.subviews
+        metadataContainer?.removeAllArrangedSubviews()
+        metadataContainer?.addArrangedSubviews((metadataSubviews?.reversed())!)
```

Comment:
Probably worth to comment what are these lines doing 👍

Reply:
I agree with you. Fixed this by adding descriptive comments regarding what those lines are doing.